[docs] Fix use of callouts (#38747)
mui/material-ui: docs/data/material/guides/minimizing-bundle-size/minimizing-bundle-size.md

The package published on npm is **transpiled**, with [Babel](https://github.com/babel/babel), to take into account the [supported platforms](/material-ui/getting-started/supported-platforms/).

:::error

> Developers are **strongly discouraged** to import from any of the other bundles directly.

Reviewer: ```suggestion
Developers are **strongly discouraged** from importing from any of the other bundles directly.
```

> Otherwise it's not guaranteed that dependencies used also use legacy or modern bundles.

Reviewer: ```suggestion
There's no guarantee that the dependencies also use legacy or modern bundles.
```

> Instead, use these bundles at the bundler level with e.g [Webpack's `resolve.alias`](https://webpack.js.org/configuration/resolve/#resolvealias):

Reviewer: ```suggestion
Instead, use these at the bundler level—for example, with [Webpack's `resolve.alias`](https://webpack.js.org/configuration/resolve/#resolvealias):
```
